hledger-lib/Hledger/Read/Common.hs

```diff
@@ -297,9 +297,19 @@ parseAndFinaliseJournal' parser iopts f txt = do
           runFin reorder ignore = journalFinalise t f txt reorder ignore
```

Comment:
Did you mean to stop calling journalFinalise here, and drop that function ?

Reply:
I was just adding it to the unprimed version as an example. If it seems like a good idea, I'll go through, add docs to the smaller functions, and integrate it into the primed version. (BTW, I still don't understand the primed one -- it seems to relate to ExceptT error catching, but I note that the program compiles fine with all tests passing if it's replaced in TimeClock and TimeDot with the unprimed one.)

Reply:
Oh -- oops. I see what you mean. One moment.